ios/src/platform_ios.mm

```diff
@@ -153,20 +153,18 @@ bool isContinuousRendering() {
 
     for (id fallback in fallbacks) {
 
-        UIFont* font = [UIFont fontWithName:fallback size:1.0];
+        handles.emplace_back([fallback]() {
 
-        size_t dataSize = 0;
-        auto cdata = loadUIFont(font, &dataSize);
+            std::vector<char> data;
 
-        if (!cdata) { continue; }
-
-        FontSourceHandle fontSourceHandle = [cdata, dataSize](size_t* _size) -> unsigned char* {
-            *_size = dataSize;
-
-            return cdata;
-        };
-
-        handles.push_back(fontSourceHandle);
+            UIFont* font = [UIFont fontWithName:fallback size:1.0];
+            size_t dataSize = 0;
+            auto cdata = loadUIFont(font, &dataSize);
```

Comment:
yeah - I thought so too, and did it, but loadUIFont is used below where it needs `unsigned char'*`. In a separate commit this can be changed to vector in both cases. 

In any way copying some memory is unlikely to take more than a few ms.